src/Query/util.ts

```diff
@@ -67,19 +69,28 @@ export const extractFacetFilters = (searchParams: URL['searchParams']): Filter<D
     const drilldownKeyIdx = key.indexOf('>');
     const baseFacet = drilldownKeyIdx > -1 ? key.slice(0, drilldownKeyIdx) : key;
 
-    queryParamLists[baseFacet] = queryParamLists[key] || {
+    queryParamLists[baseFacet] = queryParamLists[baseFacet] || {
       compound: {
         should: [],
       },
     };
+
+    // if subproduct, remove parent
+    if (drilldownKeyIdx > -1) {
+      const parent = key.slice(drilldownKeyIdx + 1, key.indexOf('>', drilldownKeyIdx + 1));
+      const indexOfParent = queryParamLists[baseFacet].compound.should.findIndex(
+        (element: Phrase) => element.phrase.query === parent
+      );
+      if (indexOfParent > -1) queryParamLists[baseFacet].compound.should.splice(indexOfParent, 1);
```

Comment:
This code block assumes that the parent facet is always going to come before the subproduct. While testing nested subfacets in Snooty it seems like this will always be the case (correct me if i'm wrong) - however, this might not always be the case when locally testing `docs-search-transport` since facets are manually being added to the request.

Reply:
good catch! but i think its safe to say that whatever is passed onto the request should be handled as an OR.

ie. `facets.target_product=atlas&facets.target_product=drivers` should come out to two should clauses.

on the other hand, if the client makes the request of a subfacet `facets.target_product=atlas&facets.target_product=drivers&facets.target_product>atlas_subproduct=atlas-cli`, that is an erroneous request since all sub products of atlas would be included.

TLDR we can remove the base facet logic and include everything in the requested facets.

if the client is including both parent and child facet (when only selecting the child facet), then we should address at client side

Reply:
also a good chance to add unit tests to above!